buildkite/providers.go

```diff
@@ -84,6 +86,23 @@ type GitHubSettings struct {
 
 func (s *GitHubSettings) isProviderSettings() {}
 
+
```

Comment:
Minor detail; `gofmt` doesn't seem to want this extra line. You might want to add https://atom.io/packages/go-plus or similar to your Atom setup, so that `gofmt` or `goimports` runs automatically.